desktop/windows/release-notes/index.md

```diff
@@ -34,6 +34,47 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 ### Security
 
+- Fixed [CVE-2022-23774](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-23774) where arbitrary files could be moved on the system.
```

Comment:
Do we normally credit the reporters of vulnerabilities in the release notes? (Two independent reports in this case). Or only in the CVE?

Reply:
I don't think we've credited the reporters of vulnerabilities in previous release notes. We link to the CVEs for details.